setup.cfg

```diff
@@ -71,6 +73,7 @@ console-scripts = weasyprint = weasyprint.__main__:main
 [options.package_data]
 weasyprint = VERSION
 weasyprint.tests = resources/*.*, resources/*/*
+weasyprint.tests.test_draw = results/*.*
```

Comment:
I think that this folder shouldn’t be included, does it?

Reply:
I’ve merged your PR and removed this line. Thank you!

Reply:
Sorry for answering so late. I thought the `options.package_data` line `setup.cfg` was necessary but I was probably mistaken so I'm glad you fixed my PR.